cross/Makefile

```diff
@@ -12,16 +12,22 @@ CARGO_PARAMS = --no-default-features --features bls12_377
 
 all: android ios
 
+android-setup:
+	rustup target add {aarch64-linux-android,armv7-linux-androideabi,arm-linux-androideabi,i686-linux-android,x86_64-linux-android}
+
 android-build: $(ARCHS_ANDROID)
 
-ios: $(LIB)
+ios-setup: 
+	rustup target add {armv7-apple-ios,armv7s-apple-ios,aarch64-apple-ios}
```

Comment:
Should switch here to Rust 1.41, since 32-bit iOS is not supported on Rust > 1.41